addons/hr_timesheet/views/project_project_views.xml

```diff
@@ -65,11 +65,10 @@
                 </xpath>
                 <xpath expr="//div[hasclass('o_project_kanban_boxes')]" position="after">
                     <t t-set="badgeColor" t-value="'border-success'"/>
-                    <t t-set="badgeColor" t-value="'border-warning'" t-if="record.remaining_hours.raw_value / record.allocated_hours.raw_value &lt; 0.2 and record.remaining_hours.raw_value / record.allocated_hours.raw_value &gt; 0"/>
-                    <t t-set="badgeColor" t-value="'border-danger'" t-elif="record.remaining_hours.raw_value &lt; 0"/>
+                    <t t-set="badgeColor" t-value="'border-danger'" t-if="record.remaining_hours.raw_value &lt; 0"/>
                     <t t-set="title" t-value="'Remaining days'" t-if="record.encode_uom_in_days.raw_value"/>
                     <t t-set="title" t-value="'Remaining hours'" t-else=""/>
-                    <div t-if="record.allow_timesheets.raw_value and record.allocated_hours.raw_value &gt; 0"
+                    <div t-if="record.allocated_hours.raw_value &gt; 0"
```

Comment:
Can you explain why did you do that? 🧐 🙂 

I did not find the point for that. 😇

Reply:
> Can you explain why did you do that? monocle_face slightly_smiling_face
> 
> I did not find the point for that. innocent


I have reverted this because there is no mentioned  in task about this. I discussed with @adjo-odoo.he told me that this changes made by mistake.thanks